### PR TITLE
Add simple client

### PR DESCRIPTION
**Pull request by Hernqvist**
### Test Plan
Tested with latest backend from master.

### Description
Add simple frontend UI that displays data from the API. Accessed through index.html

### CODE REVIEW

File: client/index.html [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+</head>
+<body>
+	<h1>
+		SjChat
+	</h1>
+	<script>
+		var apiAddress = window.location.protocol +'//'+ window.location.hostname +':8080';
+		var dataManager = new DataManager();
+
+		$.ajaxSetup({
```
Comment (s4wny) [inline client/index.html]: These lines can be removed. `async: false` is the default behaviour.

File: client/index.html [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+	<script src="js/dataManager.js"></script>
+    <script src="https://unpkg.com/react@15/dist/react.js"></script>
+    <script src="https://unpkg.com/react-dom@15/dist/react-dom.js"></script>
+    <script src="js/react/browser.min.js"></script>
+</head>
+<body>
+	<h1>
+		SjChat
+	</h1>
+	<script>
```
Comment (s4wny) [inline client/index.html]: It's better to add all scripts in separate files and include them with `<script src="js/file.js">`. The HTML and JS code becomes more readable this way.

File: client/index.html [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+
+				for (let message of messages){
+					dataManager.putMessage(chatId, message);
+				}
+			});
+		}
+
+	</script>
+
+	<div style="width: 30%; background-color: #FF55AA; float: left">
```
Comment (s4wny) [inline client/index.html]: For now it's okay to inline the css. But in the future we should not use any inlined css at all. I created an issue for this #77.

File: client/index.html [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+	<div style="width: 70%; background-color: #AA55FF; float: right">
+		<div id = "chatView"></div>
+	</div>
+
+	<div style="clear:both;">
+		<br/>
+		<p id="datadump"> </p>
+	</div>
+
+	<script type="text/babel">
```
Comment (s4wny) [inline client/index.html]: I can't see that we include babel. Do react include babel?

File: client/index.html (hunk 1 above)
Comment (s4wny) [inline client/index.html]: Line 20-23 `$.ajaxSetup({`

Comment (Hernqvist) [inline client/index.html]: The lines seem to be necessary, when the lines are removed async is on. In fact, it complains that synchronous requests are deprecated.

File: client/index.html (hunk 3 above)
Comment (Hernqvist) [inline client/index.html]: Completely agree, the design is currently made purposely low-effort.

File: client/index.html (hunk 1 above)
Comment (s4wny) [inline client/index.html]: Oh my bad. I thought it said `async: true`. We should definitely use `async: true` which is the default behaivour. With `async: false` each AJAX request will block all other javascript execution.

Also, syncrohnized requests are as you said deprecated.

File: client/index.html (hunk 4 above)
Comment (Hernqvist) [inline client/index.html]: Maybe? I'm not including it elsewhere.

File: client/index.html (hunk 1 above)
Comment (Hernqvist) [inline client/index.html]: Agree. However, I think we should add that as an issue and push this as MVP.
@s4wny

File: client/index.html (hunk 2 above)
Comment (Hernqvist) [inline client/index.html]: Done.

File: client/index.html (hunk 4 above)
Comment (Hernqvist) [inline client/index.html]: Oh, I got it, browser.min.js includes babel. @s4wny

**Pull request merged**